src/dists.ad.js

```diff
@@ -517,6 +517,44 @@ var DiagCovGaussian = makeDistributionType({
   }
 });
 
+function laplaceSample(mu, b) {
+  // Generated from goo.gl/3BxCGd (wiki)
+  var z = util.random();
+  var u = (1 - z) * -0.5 + z * 0.5;
+  return mu - b * Math.sign(u) * Math.log(1 - 2 * Math.abs(u));
+}
+
+function laplaceScore(mu, b, x) {
+  'use ad';
+  return -1 * (Math.log(2 * b) + Math.abs(x - mu) / b);
+}
+
+var Laplace = makeDistributionType({
+  name: 'Laplace',
+  desc: 'Distribution over ``[-Infinity, Infinity]``',
+  params: [{name: 'mu', desc: 'mean', type: types.unboundedReal},
+           {name: 'b', desc: 'scale', type: types.positiveReal}],
```

Comment:
Discussed on slack. Changed for Laplace, left the other ones, though I could easily change them for consistency if this is the new convention